.travis.yml

```diff
@@ -1,11 +1,10 @@
 language: node_js
 sudo: false
 install:
-  - npm install npm@2
-  - ./node_modules/.bin/npm install
+  - npm install -g npm@2
```

Comment:
Is there a reason we need a global install of `npm` here? Is this related to module caching? Could the global installation interfere with any other builds on Travis?